packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -25,7 +25,10 @@ export const PaymentConfirmation = ({
 }: PaymentConfirmationProps) => {
   const { amount, currency, interval, interval_count } = selectedPlan;
   const { displayName, email } = profile;
-  const { brand, last4, subscriptions } = customer;
+
+  const { brand, last4, payment_provider, subscriptions } = customer;
+  const isPaypalPayment = payment_provider === 'paypal';
```

Comment:
Would be nice if we have this in a lib somewhere.  Currently we have some logic checking to see if a customer has an existing CC that's duplicated in two places; preferably we don't do that with this too.